src/sentry/issues/analytics.py

```diff
@@ -7,4 +7,14 @@ class IssueForecastSaved(analytics.Event):
     attributes = (analytics.Attribute("num_groups"),)
 
 
+class IssueEscalating(analytics.Event):
+    type = "issue.escalating"
+
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("project_id"),
+        analytics.Attribute("group_id"),
+    )
+
+
```

Comment:
Any reason why you're not registering your event?

Reply:
thank you, I forgot to do this

Reply:
Did you forget to push your commit?

Reply:
yes I did, my bad - thanks again